ci-operator/step-registry/ipi/install/post-install/ipi-install-post-install-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+# if the cluster profile includes an insights secret, install it to the cluster to
+# report support data from the support-operator
+if [[ -f "${CLUSTER_PROFILE_DIR}/insights-live.yaml" ]]; then
+  oc create -f "${CLUSTER_PROFILE_DIR}/insights-live.yaml" || true
```

Comment:
Also, maybe we should take the opportunity, make this explicit and not depend on the secret being in the cluster profile bundle? We could sync it from BW to a separate secret in `test-credentials` and use `credentials:` to pull this in?

Reply:
Sounds good, but `credentials:` are creating the secret in the namespace. We need this secret in the created cluster.

Reply:
Also, we will have to make sure that this step will use the same path with the mountPath of the credentials. Maybe we can validate this|?

Reply:
What I meant is that this step relies on the `insights-live.yaml` being present in the cluster profile secret bundle. We have a first class support for passing custom secrets to steps that need them, so we can get rid of that, and just use `credentials:` to make this secret available to the step (which then takes is and creates the secret in the cluster).